core/src/main/java/com/scalar/db/storage/dynamo/DynamoAdmin.java

```diff
@@ -47,32 +136,189 @@ public DynamoAdmin(DynamoConfig config) {
     this.namespacePrefix = namespacePrefix.map(n -> n + "_");
   }
 
+  @VisibleForTesting
+  DynamoAdmin(
+      DynamoDbClient dynamoDbClient,
+      ApplicationAutoScalingClient applicationAutoScalingClient,
+      DynamoTableMetadataManager metadataManager,
+      Optional<String> namespacePrefix) {
+    this.client = dynamoDbClient;
+    this.applicationAutoScalingClient = applicationAutoScalingClient;
+    this.metadataManager = metadataManager;
+    this.namespacePrefix = namespacePrefix;
+  }
+
   @Override
-  public void createNamespace(String namespace, Map<String, String> options)
-      throws ExecutionException {
-    throw new UnsupportedOperationException("implement later");
+  public void createNamespace(String namespace, Map<String, String> options) {
+    LOGGER.info(
+        "In Dynamo DB storage, namespace will be added to table name as prefix along with dot separator.");
```

Comment:
I don't think we need logging here. Can you please just add a comment?

Reply:
Apply in [2f5e335](https://github.com/scalar-labs/scalardb/pull/301/commits/2f5e3350aab24ce146539e84efef99fc063d8769)